test/regress/regress0/echo.smt2

```diff
@@ -0,0 +1,16 @@
+; EXPECT: "str"""
+; EXPECT: "str""ing"
+; EXPECT: "hi"
+; EXPECT: "str""""ing"
+; EXPECT: "str""i""ng"
+; EXPECT: "str\ing"
+; EXPECT: "str\\ing"
+; EXPECT: "str\"
```

Comment:
Can you add one more test here: `(echo "\u{65}")`, `\u{...}` is a string literal escape sequence, which I guess should not be processed and hence printed back verbatim.

Reply:
Done